pkg/minikube/assets/addons.go

```diff
@@ -208,6 +208,19 @@ var Addons = map[string]*Addon{
 		"GlusterfsServer":        "docker.io",
 		"GlusterfileProvisioner": "docker.io",
 	}),
+	"storage-provisioner-rancher": NewAddon([]*BinAsset{
+		MustBinAsset(addons.StorageProvisionerRancherAssets,
+			"storage-provisioner-rancher/storage-provisioner-rancher.yaml.tmpl",
+			vmpath.GuestAddonsDir,
+			"storage-provisioner-rancher.yaml",
+			"0640"),
+	}, false, "storage-provisioner-rancher", "3rd party (Rancher)", "", "", map[string]string{
+		"LocalPathProvisioner": "rancher/local-path-provisioner:v0.0.22",
+		"Helper":               "busybox:stable",
```

Comment:
For security reasons these images should be pinned to a SHA, see others for example

Reply:
yes good call ! please do that @presztak you can see example of other addons 
and to get the sha you can do "docker pull" and it will spit out the sha in the end.

Reply:
@spowelljr thanks for review! Actually SHA was present in original PR but then I've realized that images with hardcoded SHA would not work on arm architecture. Should we detect architecture and regarding this use image with proper SHA or can we solve this with different approach?

Reply:
The SHA will work on arm64 as it gives the SHA of the manifest, then the manifest directs it to download the correct arch. To confirm I just tried on an amd64 & arm64 machine

amd64:
```
Digest: sha256:e34c88ae0affb1cdefbb874140d6339d4a27ec4ee420ae8199cd839997b05246
Status: Downloaded newer image for rancher/local-path-provisioner:v0.0.22
docker.io/rancher/local-path-provisioner:v0.0.22
```

arm64:
```
Digest: sha256:e34c88ae0affb1cdefbb874140d6339d4a27ec4ee420ae8199cd839997b05246
Status: Downloaded newer image for rancher/local-path-provisioner:v0.0.22
docker.io/rancher/local-path-provisioner:v0.0.22
```

Reply:
@spowelljr Done.